app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
```

Comment:
- [ ] Kindly ensure your design follows the given design [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) Kindly notice how the `login` button differs in your design from the given wireframe. Kindly fix this

Given design
![image](https://github.com/biruk-bereka/Blogify/assets/81932478/29dcf852-0537-4fa1-a3cb-24bf38988424)

Your design
![image](https://github.com/biruk-bereka/Blogify/assets/81932478/7cdb47a9-c2b3-46ec-a12d-9bfa5adacd72)